mw-webapp/src/component/image/Image.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styles from "src/component/image/Image.module.scss";
+
+interface ImageProps {
+  /**
+   * Image source
+   */
+  src: string;
+  /**
+   * Image alt text
+   */
+  alt: string;
+  /**
+   * Additional custom class name for the component
+   */
+  className?: string;
+}
+
+export const Image: React.FC<ImageProps> = (props: ImageProps) => {
```

Comment:
Please read docs again, all public members should be documented (all exported functions, variables etc.
)